src/main/java/com/example/javaOjt/services/AuthorService.java

```diff
@@ -75,4 +75,16 @@ public GetAuthorsResponse getAuthorsList(@Nullable AuthorSortBy attribute, @Null
     return new GetAuthorsResponse(authorRepository.getAuthorsList(attribute, order, ids));
   }
 
+  /**
+   * Saves a new author to the DB with the given String as the name.
+   *
+   * @param name the name of the author to be added
+   * @return the Author that is saved to the DB
+   */
+  public GetAuthorResponse postAuthor(String name) {
```

Comment:
今気づいたのでコメントしました。

service 層のメソッドに `@Transactional` をつけるのは一般的です。 DBへの処理を失敗する際には失敗した処理だけ rollback するより全ての処理を rollback するの方はよくやる対応です。

```suggestion
  @Transactional
  public GetAuthorResponse postAuthor(String name) {
```

Reply:
@b02902092 なるほど。DBへの処理がある場合は、Serviceに@Transactionalつけるんですね！覚えておきます。